packages/now-static-build/src/index.ts

```diff
@@ -530,9 +537,16 @@ export async function build({
     );
     const spawnOpts = getSpawnOptions(meta, nodeVersion);
     await runShellScript(path.join(workPath, entrypoint), [], spawnOpts);
-    validateDistDir(distPath);
+
+    const extraOutputs = await readBuildOutputDirectory({ workPath });
```

Comment:
Are you sure we want to enable this feature for legacy deployments?

Seems like it should only be enabled for `zeroConfig: true`.

Reply:
I don't think it'll matter, but yea, no need to change the legacy path.